src/impl/realm_coordinator.cpp

```diff
@@ -241,6 +237,9 @@ std::shared_ptr<Realm> RealmCoordinator::get_realm(Realm::Config config)
         m_weak_realm_notifiers.emplace_back(realm, m_config.cache);
     }
 
+    if (realm->config().sync_config)
```

Comment:
missed `#if REALM_ENABLE_SYNC` here

Reply:
The `sync_config` member of `Realm::Config` is available even when sync is not enabled, so AFAICT no `#if` is needed here?

Reply:
but `create_sync_session();` is not there if sync is not enabled, right?

Reply:
It is: https://github.com/realm/realm-object-store/blob/90d7e73ad4605618c414e1d517bb83a4229e0538/src/impl/realm_coordinator.cpp#L78

Reply:
uh, sorry!! I was thinking a wrong thing `SyncSession::create_sync_session()`. 

👍